<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberSelectWindow</class>
 <widget class="QMainWindow" name="NumberSelectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请选择菜品数量</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numberSelectSpinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="numberSelectConfirmButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="numberSelectCancelButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
